src/general.rs

```diff
@@ -249,16 +250,23 @@ pub fn initialize(
                     token_types: ctx
                         .config
                         .semantic_tokens
-                        .keys()
+                        .iter()
                         .cloned()
-                        .map(|x| x.into())
+                        .map(|token| token.name.into())
```

Comment:
We don't need to used `cloned()` anymore since we don't need to clone the entire `SemanticTokenConfig` object.
Instead something like `.map(|token| token.name.clone().into())` would work. Same below.